src/Modal/Modal.jsx

```diff
@@ -1,20 +1,37 @@
-import React from "react";
+import React, { Component } from "react";
 import PropTypes from "prop-types";
-import ModalStyles from "./modal.style";
+import { disableBodyScroll, clearAllBodyScrollLocks } from "body-scroll-lock";
+import { ModalStyles, Overlay, Content } from "./modal.style";
 
-export default function Modal({ open, children }) {
-  return (
-    <ModalStyles>
-      {open ? (
-        <div className="container">
-          <div className="modal-body">{children}</div>
-        </div>
-      ) : null}
-    </ModalStyles>
-  );
+class Modal extends Component {
+  componentDidMount() {
+    disableBodyScroll(document.querySelector("body"));
+  }
+
+  componentWillUnmount() {
+    clearAllBodyScrollLocks();
```

Comment:
Better to call `enableBodyScroll` on the same element that you called disable on here (`body`) rather than disable all scroll locks as you might inadvertently disable someone elses